src/vm-repair/azext_vm_repair/custom.py

```diff
@@ -164,6 +167,32 @@ def create(cmd, vm_name, resource_group_name, repair_password=None, repair_usern
                                   .format(g=repair_group_name, disk_name=copy_disk_name, vm_name=repair_vm_name, uri=copy_disk_id)
             _call_az_command(attach_disk_command)
 
+        # invoke enable-NestedHyperV.ps1 again to attach Disk to Nested
+        if enable_nested:
+            logger.info("Running hyperv")
+
+            stdout, stderr = _invoke_run_command("win-enable-nested-hyperv.ps1", repair_vm_name, repair_group_name, 0)
```

Comment:
Does it support Linux environment?

Reply:
No, this is only supported on windows. I will add the check in validation.

Reply:
Added validation.